src/main/java/it/pagopa/interop/probing/eservice/operations/util/FunctionContributor.java

```diff
@@ -11,8 +11,8 @@ public class FunctionContributor implements MetadataBuilderContributor {
 
   @Override
   public void contribute(MetadataBuilder metadataBuilder) {
-    metadataBuilder.applySqlFunction("make_interval",
-        new SQLFunctionTemplate(TimestampType.INSTANCE, "?1 + MAKE_INTERVAL(mins => ?2)"));
+    metadataBuilder.applySqlFunction("make_interval", new SQLFunctionTemplate(
+        TimestampType.INSTANCE, "DATE_TRUNC('minute',?1) + MAKE_INTERVAL(mins => ?2)"));
```

Comment:
truncation should be done in the service that specifically needs this logic.
`operations` should send the value as is, because different services in the future may use this value with different requirements

Reply:
this sql function is used only for the get query of the eservices ready for polling.The trunc was made for an issue that we discussed on Friday.

Reply:
ok, I thought this was returned also in `getEserviceMainData`